src/translations/sqlb_zh.ts

```diff
@@ -214,67 +214,68 @@
         <location filename="../Application.cpp" line="93"/>
         <source>Usage: %1 [options] [&lt;database&gt;|&lt;project&gt;]
 </source>
-        <translation type="unfinished"></translation>
+        <translation>用法: %1 [选项] [&lt;数据库&gt;|&lt;项目&gt;]
+</translation>
     </message>
     <message>
         <location filename="../Application.cpp" line="96"/>
         <source>  -h, --help          Show command line options</source>
-        <translation type="unfinished"></translation>
+        <translation>  -h, --help          显示命令行选项</translation>
     </message>
     <message>
         <location filename="../Application.cpp" line="97"/>
         <source>  -q, --quit          Exit application after running scripts</source>
-        <translation type="unfinished"></translation>
+        <translation>  -q, --quit          在运行脚本后退出应用程序</translation>
     </message>
     <message>
         <location filename="../Application.cpp" line="98"/>
         <source>  -s, --sql &lt;file&gt;    Execute this SQL file after opening the DB</source>
-        <translation type="unfinished"></translation>
+        <translation>  -s, --sql [文件]    在打开数据库后执行此 SQL 文件</translation>
```

Comment:
Just curious, Why did you use [brackets] instead of the <> symbols?

Reply:
Sorry I didn't notice the original text didn't use the brackets

Reply:
Fine. Not much of a problem. :smile: